.github/workflows/validate-renovate.yml

```
@@ -0,0 +1,31 @@
+name: 'Validate renovate config'
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'env/renovate.json'
```

Should this be `.github/rennovate.json`? I'm not understanding why it changes here.
```suggestion
      - '.github/renovate.json'
```